cmakeModules/FindROOT.cmake

```diff
@@ -181,35 +175,19 @@ else()
 	string(REGEX REPLACE "^[0-9]+\\.[0-9][0-9]+\\/([0-9][0-9]+).*" "\\1"
 		ROOT_PATCH_VERSION "${ROOT_VERSION}")
 	set(ROOT_VERSION "${ROOT_MAJOR_VERSION}.${ROOT_MINOR_VERSION}.${ROOT_PATCH_VERSION}")
-	# compare version
-	if(ROOT_FIND_VERSION_EXACT)
```

Comment:
Is there a reason to drop this functionality?

Reply:
This is handled by `find_package_handle_standard_args`, i.e. `find_package(ROOT 6.08.00 EXACT ...` should work. Didn't try it though...